src/pages/browse.jsx

```diff
@@ -37,29 +37,42 @@ const ProgressIndicator = (
 );
 
 let fridgeList = null;
+const fridgePaperBoyLoveGallery = [40.697759, -73.927282];
+let prevFocusedMarker = null;
```

Comment:
These two variables look like they're only used inside of the BrowsePage component. Is there a reason for them to be declared globally to the file instead of locally to the component that uses them?

Reply:
the first one is just meant to initiate the state, and I didn't want to do useState([40.697759, -73.927282]), and it's a practice of mine to keep things that initiate state outside of the component.
for the second one it's just choice between useRef and simple var